pkg/framework/webhooks.go

```diff
@@ -97,7 +97,7 @@ func UpdateMutatingWebhookConfiguration(c client.Client, updated *admissionregis
 
 // UpdateValidatingWebhookConfiguration updates the specified mutating webhook configuration
 func UpdateValidatingWebhookConfiguration(c client.Client, updated *admissionregistrationv1.ValidatingWebhookConfiguration) error {
-	return wait.PollImmediate(RetryShort, WaitShort, func() (bool, error) {
+	return wait.PollImmediate(RetryShort, WaitMedium, func() (bool, error) {
```

Comment:
can you elaborate what's motivating this change? can you share CI failures links that support that motivation and that this change would mitigate?

Reply:
I can't but shortly the workflow I imagine:webhook is deleted,

Reply:
>https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-api-actuator-pkg/183/pull-ci-openshift-cluster-api-actuator-pkg-master-e2e-aws-operator/1290937797384867840

mm that link happened running against this PR... And the failure is for `API operator deployment should [Serial] maintains deployment spec`, so I'm not really sure how it's related to this change at all.

Those links are timeouts, if they are match the something went wrong with the operator, none test will pass. Some time to run the pod + the 3 min to check permanent availability should be the norm.

```UpdateValidatingWebhookConfiguration``` are atomic functions, the only reason they have a PollImmediate atm is to mitigate transient errors and should probably top doing that, assume there won't be any and if there's we fail the test. If something needs to wait for a good reason please create a `WaitBlah` function.

Reply:
That's why enabling `Serial` keyword is necessary here. It failed on this line, and that's the case I described:

https://github.com/openshift/cluster-api-actuator-pkg/blob/e943d397dea02e9c3f09cb1dc6543cf4a0303da0/pkg/operators/machine-api-operator.go#L60

> But shortly the workflow I imagine is happening:
> 
>     * tests are running in parallel
> 
>     * webhook is deleted
> 
>     * MAO is busy on other things, and does not yet know (1-3 minutest), with the inclusion of [openshift/machine-api-operator#651](https://github.com/openshift/machine-api-operator/pull/651) it is now up to 10 minutes (https://github.com/openshift/machine-api-operator/blob/6e867bf92c3ec296c4c93d73abd93b47af9426a3/pkg/operator/sync.go#L31 + https://github.com/openshift/machine-api-operator/blob/6e867bf92c3ec296c4c93d73abd93b47af9426a3/pkg/operator/sync.go#L28)

Reply:
In order to enable this, we need to merge https://github.com/openshift/cluster-api-actuator-pkg/pull/186 first.